conjur/logic/credential_provider/file_credentials_provider.py

```diff
@@ -166,3 +147,25 @@ def _log_netrc_warning(cls):
                             f"plaintext in '{DEFAULT_NETRC_FILE}'. Make sure to logoff "
                             "after you have finished using the CLI")
             cls.FIRST_TIME_LOG_INSECURE_STORE_WARNING = False
+
+    def _get_credentials_from_file(self, conjurrc_conjur_url):
```

Comment:
as discussed internal classes don't get UTs so add no-pragma if we don't want it to count in our code coverage percentage

Reply:
done